### DISCUSSION TITLE
How to buy this suite

### DISCUSSION THREAD

**Discussion by maniandaji88** [oliver-zehentleitner/unicorn-binance-websocket-api#384] [Q&A] [answered]
Hello, I find the code is open-source, but you said the ‘UNICORN Binance Suite’ need a license, it means I need to pay for it. So I am confused, if I only pip this library in my laptop, then use it, so how to pay for it. If I pay for it, what I can obtain? Looking forward to your answer.

**Comment by oliver-zehentleitner** [comment, marked as answer]
Hello!

Open source means that it is openly available as source code. This creates transparency for the user (i.e. for you) and helps to improve the product.

You can install it via PIP and get a compiled C-Extention. To run it, you need a license, if you don't have it, the library will throw a Python exception when loading.

You can test it with a free 7 day license:
https://shop.lucit.services/software/p/unicorn-binance-suite-test-license-7-days

**Comment by maniandaji88** [comment]
I think some code is not open for everyone, otherwise, one can directly use it and do not pay for it. Is my guess correct?

**Comment by oliver-zehentleitner** [comment]
No, that is not correct. 

https://www.redhat.com/en/topics/open-source/what-is-open-source-software

This is our license: https://lucit-licensing-python.docs.lucit.tech/license.html
